docs/docs/tutorials/cost-estimate-report.md

```diff
@@ -0,0 +1,191 @@
+
+## Generating cost estimate
+
+To generate the cost estimate we will use the following command:
+
+```sh
+nebari cost-estimate
+```
+
+The `nebari cost-estimate` will look for the `stages` directory and analyze the Terraform templates to generate a cost report. It will consist of two tables describing the cost and resources breakdown, a publicly-accessible Infracost dashboard URL and additional documentation on the edge cases that the cost estimate tool currently misses.
+
+It will be visible in the following manner:
+
+```sh
+         Cost Breakdown          
+┏━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┓
+┃               Name ┃ Cost ($) ┃
+┡━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━┩
+│ Total Monthly Cost │       XX │
+│  Total Hourly Cost │       XX │
+└────────────────────┴──────────┘
+           Resource Breakdown           
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┓
+┃                         Name ┃ Number ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━┩
+│         Total Detected Costs │      X │
+│    Total Supported Resources │      X │
+│ Total Un-Supported Resources │      X │
+│   Total Non-Priced Resources │      X │
+│ Total Usage-Priced Resources │      X │
+└──────────────────────────────┴────────┘
+Access the dashboard here: 
+https://dashboard.infracost.io/share/XXXXXXXXXX
+
+QHub rely upon node-pools which is a usage resource but doesn't get captured in 
+the above report. A general node-pool will always have one node running will add
+quite an additional charge. Please check in with your cloud provider to see the 
+associated costs with node pools.                                               
+
+ • Total Monthly Cost: The total monthly cost of the deployment of supported    
+   resources.                                                                   
+ • Total Hourly Cost: The total hourly cost of the deployment of supported      
+   resources.                                                                   
+ • Total Detected Costs: The total number of resources detected by Infracost.   
+ • Total Supported Resources: The total number of resources supported by        
+   Infracost.                                                                   
+ • Total Un-Supported Resources: The total number of resources unsupported by   
+   Infracost.                                            
+ • Total Non-Priced Resources: The total number of resources that are not       
+   priced.                                               
+ • Total Usage-Priced Resources: The total number of resources that are priced  
+   based on usage.                                                              
+```
+
+You can share the URL of your Infracost dashboard `https://dashboard.infracost.io/share/XXXXXXXXXX
+` across your team for better insights into your cloud costs breakdown.
+
+If your `stages` subdirectory is not present, you would need to share the path of your `stages` directory with the `--path` option.
```

Comment:
What is the `stages` subdirectory? Why wouldn't I have one?

Reply:
This is where the Terraform templates for deployment are available — It might be possible that the user wants to run the cost estimate tool in a different directory than the one they are present in right now.

Reply:
I still don't understand why I wouldn't have a stages directory. Maybe this can be clarified by saying: 

>You can run the Nebari cost estimate tool outside of your Nebari directory by specifying the location of your deployment's `stages` directory.